packages/debug/src/browser/debug-session-contribution.ts

```diff
@@ -114,6 +115,8 @@ export class DefaultDebugSessionFactory implements DebugSessionFactory {
     protected readonly debugPreferences: DebugPreferences;
     @inject(FileSystem)
     protected readonly fileSystem: FileSystem;
+    @inject(RunInTerminalUtility)
+    protected readonly runInTerminalUtility: RunInTerminalUtility;
```

Comment:
looks like this file can be reverted

Reply:
Yes, I pushed my changes mid-edition yesterday evening, I was planning on finishing today :)